source/class/cv/plugins/diagram/AbstractDiagram.js

```diff
@@ -361,7 +361,12 @@ qx.Class.define('cv.plugins.diagram.AbstractDiagram', {
     },
 
     _onStatusError: function(ts, key, ev) {
-      qx.log.Logger.error(this, '_onStatusError', ts, key, ev);
+      cv.core.notifications.Router.dispatchMessage('cv.diagram.error', {
+        title: 'Diagram communication error',
```

Comment:
Texts shown to the user should be made translatable and then translated to german.